docs-chatbot/tsconfig.json

```
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

```suggestion
    "target": "es2017",
```

In the tutorial, we leave this at the default, use an IISE to wrap the function we need to await, and explain you can set a different target to avoid having to do that. Let's just set that target, explain briefly why in the tutorial (I can handle this.), and then we can avoid having to explain the IISE and tell how you can avoid needing it.